Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -154,15 +154,15 @@ extension HTTPRequest.Path {
         case .getCustomerInfo,
                 .logIn,
                 .postReceiptData,
+                .getProductEntitlementMapping,
```

Comment:
Hmm don't we need to align on the mechanism for signing these requests?

Reply:
Yeah I’m doing that separately. I thought these already had signing with nonces but no, that’s what tests are failing,